stdlib/Markdown/src/render/terminal/render.jl

```diff
@@ -9,24 +9,26 @@ function term(io::IO, content::Vector, cols)
     isempty(content) && return
     for md in content[1:end-1]
         term(io, md, cols)
-        println(io)
+        print(io, '\n', '\n')
```

Comment:
Why two newllines?

Reply:
Oh I see --- because each element used to end with a newline, but those are now removed?

Reply:
Yes.